nf-test.config

```
@@ -0,0 +1,24 @@
+config {
+    // location for all nf-test tests
+    testsDir "."
+
+    // nf-test directory including temporary files for each test
+    workDir System.getenv("NFT_WORKDIR") ?: ".nf-test"
+
+    // location of an optional nextflow.config file specific for executing tests
+    configFile "tests/nextflow.config"
+
+    // ignore tests coming from the nf-core/modules repo
+    ignore 'modules/nf-core/**/*', 'subworkflows/nf-core/**/*'
```

ignore 'modules/nf-core/**/tests/*', 'subworkflows/nf-core/**/tests/*'
```suggestion
    ignore 'modules/nf-core/**/*', 'subworkflows/nf-core/**/*'
```

---

Do you mean
`ignore 'modules/nf-core/**/tests/*', 'subworkflows/nf-core/**/tests/*'`